src/main/java/org/opentripplanner/standalone/config/routerconfig/updaters/azure/SiriAzureUpdaterConfig.java

```diff
@@ -24,6 +25,16 @@ public static void populateConfig(
     parameters.setFuzzyTripMatching(
       c.of("fuzzyTripMatching").since(NA).summary("TODO").asBoolean(false)
     );
+    parameters.setFullyQualifiedNamespace(
+      c.of("fullyQualifiedNamespace").since(NA).summary("TODO").asString(null)
+    );
```

Comment:
Let me know if you're interested in producing documentation for these values. (Since it's Sandbox I will not insist on it.)

Reply:
Fixed